.gitignore

```diff
@@ -5,6 +5,8 @@
 pom.xml
 dev-config.edn
 test-config.edn
+local_protobuf.clj
+*-local.edn
 *.jar
 *.class
 /.lein-*
```

Comment:
I know we aren't including the .protos in the repo.
I forget why we decided not to add *.proto to gitignore.

Reply:
we ADO include the `person.proto` and `message.proto` files so we can't blanket ignore *.proto files